runtime/kusama/src/lib.rs

```diff
@@ -1499,6 +1518,16 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 	}
 }
 
+// Gov V1 pallets pending storage removal using the frame_support RemovePallet migration.
+parameter_types! {
+	pub const DemocracyStr: &'static str = "Democracy";
```

Comment:
wondering when static str can be used as const-generics, which would allow us to have one `ConstStr<const S: &'static str>` or something like that.